### ISSUE TITLE
`load-theme` is called twice when `ModeChanged` is signaled

### ISSUE THREAD

**Issue by grtcdr** [grtcdr/darkman.el#9]
This leads to a theme transition that isn't very smooth. I have no idea what might be
causing this, but a peek into Darkman's source code could be helpful.

**Comment by aerz** [comment]
The only difference I noticed is that Emacs is getting a signal from DBus of a `method-call` and then another signal of a `method-return`. :thinking:

**Comment by aerz** [comment]
I found a way to deal with this issue by ussing `dbus-register-monitor` to monitor a specific signal call instead `dbus-register-signal` which handles all kinds of signals. The bad thing is that requires Emacs 28.1 which is a higher version but I don't know what you think about giving greater compatibility vs. being feature-centric.

I attached a patch if you want to check it. Last thing I want to ask you is if there is a way to ignore some arguments on ` darkman--mode-changed-signal-handler ` called because now the function will receive several new info with this format: 
```
* darkman--mode-changed-signal-handler("nl.whynothugo.darkman" "Mode" ("light))
``` 

But we want just the last string list which includes the new darkman mode.

[darkman-dbus.patch.zip](https://github.com/grtcdr/darkman.el/files/10844246/darkman-dbus.patch.zip)
